src/app/application/components/app-view/AppView.vue

```diff
@@ -21,6 +21,7 @@
         <slot name="title" />
 
         <div
+          v-if="$slots.actions"
```

Comment:
How come we needed this?

Reply:
I noticed that this renders an empty element even when there are no actions slotted-in. This messes with the parent's flex layout. Generally, optional slots should be guarded like this.

Reply:
From that I understood that it wasn't needed for the upgrade?

I know this sounds like a nit, but it would be really good to get used to doing these types of PRs (upgrades, important bugfixes) as only whats needed to do the upgrade or important bugfix. That way its more likely we avoid regressions or further errors from doing an upgrade or important bugfix.

Sorry, I know its this tiny bit of code, but could this one go in another PR if poss? Also, is this currently causing an issue anywhere (no reason not to fix if not, just curious).

I get the feeling this PR isn't going in that soon as the kongponents side is still in draft, but if not I'm going to approve anyway incase you need to merge quick without removing this.

Reply:
I think this _needs_ to be removed, 99% sure. So even for the 1% that I'm wrong it doesn't hurt to remove and really it shouldn't be in this PR. Let me dismiss the approval, sorry about that

Reply:
This isn't addressing the breaking changes, and I think adding this change here creates a bug elsewhere. Could you you remove it from this PR please? We can circle back after if necessary.

Reply:
If this creates a bug somewhere then surely that needs fixing, right? Can you tell me what this could break and file an issue for it?

A hand-wavy suggestion to remove the appropriate guard on an optional slot doesn’t seem to break the idea of having these adoption PRs be fairly minimal.

Reply:
FYI I did remove this now, but I expect a better explanation for why this insistence was necessary.

Reply:
This insistence was necessary because I've been saying throughout this PR that we should just be addressing the breaking changes in the kongponents upgrade here, and the before and after screengrabs you gave above showing the line wrapping if the title is exactly a certain length with no breaking characters is not something I would cout as just addressing the breaking changes.

---

Apart from this, FYI:

https://github.com/kumahq/kuma-gui/pull/1729

<img width="854" alt="Screenshot 2024-02-23 at 10 26 24" src="https://github.com/kumahq/kuma-gui/assets/554604/bb16cd2f-f12c-4e5f-a375-bededbcfc548">